### ISSUE TITLE
Add service to insert bible passage into text-documents

### ISSUE THREAD

**Issue by filipkemuel** [mdbergmann/Eloquent#14]
It would be really cool if it would be possible to create a service for the service-menu that could insert bible passages directly into a word-processor.

Especially if it was possible to insert the verses with some styling. Like this:

<sub>1  </sub>In the beginning God created the heavens and the earth. <sub>2  </sub>And the earth was waste and void; and darkness was upon the face of the deep: and the Spirit of God moved upon the face of the waters. <sub>3  </sub>And God said, Let there be light: and there was light. <sub>4  </sub>And God saw the light, that it was good: and God divided the light from the darkness. <sub>5  </sub>And God called the light Day, and the darkness he called Night. And there was evening and there was morning, one day. (**Genesis 1:1-5**)

At the moment I just copy/paste verses from Eloquent and then add the subscript styling afterwards. But It is quite time consuming. So a feature like this would really be a time saver.

Here are examples of "services" from other bible-software:
[http://www.accordancebible.com/Accordance-Service](http://www.accordancebible.com/Accordance-Services)
[https://community.logos.com/forums/t/30583.aspx](https://community.logos.com/forums/t/30583.aspx)

**Comment by mdbergmann** [comment]
I'll look into that.

Have you had a look at Automator?

**Comment by mdbergmann** [comment]
[Services](https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/SysServices/introduction.html#//apple_ref/doc/uid/10000101-SW1)

**Comment by filipkemuel** [comment]
Yes.. Automator can be used to create the services.. but to me it seems that apps somehow needs to create some hooks that Automator can use.. 
At least I havent found a way to make automations with eloquent yet..

It might be possible with AppleScript.. But I just haven't been able to wrap my head around the syntax..

**Comment by mdbergmann** [comment]
Yeah, you could create an Automator script that will take a bible reference from the pasteboard, then you could record a script that will press: 
- CMD-SHIFT-N (to create a new bible window)
- CMD-L (to focus the reference text field in the toolbar)
- CMD-V (to paste the bible reference into the text field) and then 
- RETURN (to mate Eloquent show the text for the reference)

But I must admit this is very limited. You can't (as far as I have tested) copy back the bible text to the pasteboard.
